src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -1919,6 +1929,38 @@ private boolean isRespondentDetailsConfidential(CaseData caseData) {
         }
     }
 
+    private boolean isApplicantRepresented(CaseData caseData) {
+        if (PrlAppsConstants.C100_CASE_TYPE.equals(CaseUtils.getCaseTypeOfApplication(caseData))) {
+            return isCaApplicantRepresented(caseData);
+        } else {
+            return isDaApplicantRepresented(caseData);
+        }
+    }
+
+    public boolean isCaApplicantRepresented(CaseData caseData) {
+        Optional<List<Element<PartyDetails>>> applicantsWrapped = ofNullable(caseData.getApplicants());
+
+        if (applicantsWrapped.isPresent() && !applicantsWrapped.get().isEmpty()) {
```

Comment:
is this really required. since applicants/respondents will be Non empty..

Reply:
Not required, however we need it as sonarLints complaints about it.